test/samples_test.cpp

```diff
@@ -255,3 +255,16 @@ TEST(Samples, Sponge4) {
 #endif
 }
 #endif
+
+#ifdef MANIFOLD_EXPORT
+TEST(Samples, SelfIntersect) {
+  manifold::PolygonParams().processOverlaps = true;
+  std::string file = __FILE__;
+  std::string dir = file.substr(0, file.rfind('/'));
+  Manifold m1 = ImportMesh(dir + "/../samples/models/self_intersectA.glb");
```

Comment:
Is there a chance these are simple enough they can be generated by e.g. an Extrude and Warp? Still, there is value in being able to add test models - this may be nice as a helper function. I think I'd prefer to put them in a dedicated models directory in `test` rather than `samples` though.

Reply:
Probably not, it’s a bit sensitive and I couldn’t replicate when I tried with other self intersected tubes,
Maybe if you managed to output the exact same sweep but probably lot of trial & error, and I’m lazy.

Reply:
Okay, no prob - but let's at least change the directory.